packages/cli/package.json

```diff
@@ -8,7 +8,7 @@
     "access": "public"
   },
   "bin": {
-    "react-native": "build/bin.js"
+    "rnc": "build/bin.js"
```

Comment:
let's secure the package first. AFAIK `rnc` is taken, so temporarily I'd go for something like `rnccli` which is available

Reply:
Might also be nice to bind to `@react-native-community/cli`, just in case npm decides to break something again with `npx ...` 🙈